### ISSUE TITLE
dkim cannot be used on alias even on alias with own emails

### ISSUE THREAD

**Issue by aqueos** [virtualmin/virtualmin-gpl#779]
hi,

 If i have a website with a domain in alias the dkim cannot be set for the alias. If its a simple alias i cannot choose to use the dkim of the main domain. If it is an alias with own email i cannot either create a dkim.

 I tried alias with own email or not and both lack the dkim option.

 I think we should be able to add the domain for the dkim signing and perhaps even have its own key as there is case when alias could have a different DKIM than the main domain, especialy if it is an alias with own email or a subdomain.

 As stated in another issue i dont think DKIM should be limited to domain/alias/subdomains with email as it concern OUTGOING mail and not imap/pop and websites can also need to sign outgoing emails. But it also is lacking management of alias and alias with own emails :)

best regards,
Ghislain.

**Comment by chris001** [comment]
Good point.  Plus, DKIM signing is one essential part of making sure your outgoing emails would be deliverable to the destination inbox, not go to the spam/junk.

**Comment by jcameron** [comment]
DKIM should work for alias domains already, as long as email is enabled for the alias. 

Also in the next release we'll support DKIM for domains without email enabled, as tracked in http://github.com/virtualmin/virtualmin-gpl/issues/777

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run.

⚠️ If the issue is resolved, please take a moment to close it to help keep the project organized.
